modules/dop/initialize.go

```diff
@@ -999,3 +1001,28 @@ func compensateIssueStateCirculation(ep *endpoints.Endpoints) error {
 
 	return ep.DBClient().BatchCreateIssueTransition(statesTrans)
 }
+
+func deleteWebhook(bdl *bundle.Bundle) error {
+	const (
+		createHookName = "guide_create"
+		deleteHookName = "guide_delete"
+	)
+
+	hookNames := []string{createHookName, deleteHookName}
+
+	for _, v := range hookNames {
+		err := bdl.DeleteWebhook(apistructs.DeleteHookRequest{
```

Comment:
`deleteHookName` will never be deleted if `createHookName` deleted success but `deleteHookName` failed at first time.